FordLabs/PeopleMover
# 137 sortby redesign

## Issue
Resolves 137

## What was done
- [x] move the sortby dropdown and styled it to match new design
- [x] made the sort by drop down accessible
- [x] dropby drop down open on key down
- [x] dropby drop down can be navigated with key up and down (doesn't exit the dropdown when using arrow)
- [x] dropby dropdown support tabbing behavior
- [x] dropby dropdown close when clicking on the button that oopen it or the arrow icon
- [x] edit menu drop down is now navigatable with arrow key too
- [x] edit dropdown can be close on the icon that open it
- [x] user dropdown open on key down
- [x] user dropdown can be close by clicking on the button that open it
- [ ] IS NOT navigatable by with arrow key (Write a different story for it)

## How to test
1. Add #newui to the url to see the new design and functionality 
2. Try opening with arrow key down after tabbing on it
3. Try to navigate with tab and arrow key inside
4. Open it and close it using the sortby button
5. When selecting an option to sort by see that the option selected appear in the button text and sorting works appropriately
6. Check that product menu open on key down and can be navigated with arrow keys
7. Check that product menu open and close it using the button
8. Check that user drop down menu open on key down
9. Check that user drop down menu open and close it using the button

## Accessiblity Criteria

- [ ] Voice over is understandable and logical - [ ] Aria labels are added to elements with no visible text
